### DISCUSSION TITLE
Running Lammps parallel in conda environment installation (easy installation method)

### DISCUSSION THREAD

**Discussion by ballal09** [deepmodeling/deepmd-kit#1978] [Q&A] [answered]
Hello

I hope you are doing well.  I have installed the DeepMD kit (DeePMD-kit v2.1.4) following the easy installation procedure with conda environment.  The lammps is running with no issue in a single core but when I try to run lammps parallel, it is actually calculating the same thing multiple time.  I am using the following command to run lammps parallel:  mpirun -np 4 lmp -in in.lammps.  But it is mentioned at the beginning of the easy installation method that  mpirun can be used to run lammps parallel.  Can you please clarify how to run lammps parallel when DeepMD kit is installed via easy installation method in a conda environment.  Thanks in advance for your cooperation.

Best
Ballal

**Comment by njzjz** [comment, marked as answer]
Could you check if MPI is consistent by executing 
```sh
mpirun -h
lmp -h
```
MPI information will be printed on the screen.

**Reply by ballal09** [reply]
Thank you very much. That was helpful to figure out the  problem.
